plugins/datastore.js

```diff
@@ -33,7 +33,9 @@ const SCHEMA_SCAN = Joi.object().keys({
         keyword: Joi.string().max(200).required()
     }),
     sort: Joi.string().lowercase().valid(['ascending', 'descending']).default('descending'),
-    sortBy: Joi.string().max(100)
+    sortBy: Joi.string().max(100),
+    exclude: Joi.array().items(Joi.string()),
```

Comment:
might be better to put max as well, as in `Joi.array().items(Joi.string().max(100)),`